File: solutions/beecrowd/2003/2003.cpp
#include <cstdint>
#include <cstdio>
#include <iostream>

int main() {
    std::int16_t hour, minute, late;

    while (scanf("%d:%d", &hour, &minute) != EOF) {
        std::cout << "Atraso maximo: "
                  << (hour >= 7 ? minute : 0)
                + (hour - 7 > 0 ? (hour - 7) * 60 : 0)
                  << std::endl;
    }

    return 0;
}

File: solutions/beecrowd/2003/2003.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

while line := input().strip():
    hour, minute = map(int, line.split(':'))

    if hour < 7:
        delay = 0
    elif hour == 7:
        delay = minute
    else:
        delay = minute + (hour - 7) * 60

    output(f'Atraso maximo: {delay}\n')
